### ISSUE TITLE
Right-click on icons to open in background tab

### ISSUE THREAD

**Issue by Gitoffthelawn** [CanisLupus/swift-selection-search#194]
Hi Daniel!  I hope you are happy and healthy.

Would it be possible to implement right-clicking on SSS icons to open the item in a background tab instead of a foreground tab?

Thanks!

**Comment by CanisLupus** [comment]
Hi! Thankfully I'm in good health in these troubled times. :) I hope you and yours are too.

I've recently confirmed that it was possible to detect the mouse button that was used to click the context menu, so this seems possible. There are also a few other related requests so I'll take a look at everything when possible. There are a bunch of SSS to-dos pilling up but I'll get there in time. ;)

Cheers!

**Comment by CanisLupus** [comment]
Hey Gitoffthelawn! Since there were a few similar request I bundled them for SSS version 3.43.0 (now 3.43.2, I sincerely hope you weren't affected by the settings reset problem!).

You can now choose the left, right, and middle click behaviour for popup icons AND context menu icons (previously only left/middle click was available for popup and only left click for the context menu).

As usual, please let me know if anything is wrong. The update had quite a few changes!

Cheers!
Daniel

**Comment by Gitoffthelawn** [comment]
@CanisLupus  Thank you Daniel!!!  I took a look at what you did, and it looks great!

I'll definitely test it out too.

Fortunately, I didn't get bitten by the momentary data-loss bug.  I saw your release notes and quickly backed it up before updating... just in case! :)

The new dark mode is great.  I spent a little time in the settings tweaking everything, and I love what you've done with your fabulous extension!
